ndt-server.go

```diff
@@ -75,7 +75,7 @@ func defaultHandler(w http.ResponseWriter, req *http.Request) {
 
 <p>More information about <a href="https://measurementlab.net/tests/ndt/">NDT</a>, other M-Lab hosted <a href="https://measurementlab.net/tests/">tests</a>, the M-Lab platform, and the open data we publish can be found on our <a href="https://measurementlab.net">website</a>.</p>
 
-<p>Please visit https://measurementlab.net</a> for complete information about the M-Lab platform, tests, and data.</p>
+<p>If you need assistance, please email support@measurementlab.net, or visit <a href="https://measurementlab.net">our website</a> for complete information about the M-Lab platform, tests, and data.</p>
```

Comment:
Please change the @ sign to:
```html
	&#64;
```
which is rendered as @ but is less likely to be picked up by simple robots.  If someone has built a sophisticated robot, then they already know about webpages with that email address on them, so it's fine.

Reply:
Addressed in latest commit.